state-chain/runtime/src/lib.rs

```diff
@@ -756,6 +770,7 @@ construct_runtime!(
 	{
 		System: frame_system,
 		Timestamp: pallet_timestamp,
+		CfeEventEmitter: pallet_cf_cfe_event_emitter,
```

Comment:
Discussion here: https://discord.com/channels/775961728608895008/1195210668819878030

Reply:
Moved back to the end of the list (now using custom order of pallet execution instead)